src/main/java/com/fjb/sunrise/controllers/CategoryController.java

```diff
@@ -0,0 +1,83 @@
+package com.fjb.sunrise.controllers;
+
+import com.fjb.sunrise.dtos.requests.CategoryCreateDto;
+import com.fjb.sunrise.dtos.requests.CategoryUpdateDto;
+import com.fjb.sunrise.services.CategoryService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+@RequestMapping("/category")
+@RequiredArgsConstructor
+public class CategoryController {
+    private final CategoryService categoryService;
+
+    @GetMapping
+    public ModelAndView index() {
+        ModelAndView modelAndView = new ModelAndView();
+        modelAndView.addObject("categories", categoryService.getAllCategories());
+        modelAndView.setViewName("category/index");
```

Comment:
Consider replace this with a final variable.
For ex: private static final String CATEGORY_INDEX  = "category/index"
By reusing it in different places, maintenance becomes easier.